packages/react-instantsearch-core/src/core/__tests__/createConnector.js

```diff
@@ -723,7 +723,7 @@ describe('createConnector', () => {
 
       expect(onSearchStateChange).toHaveBeenCalledTimes(1);
       expect(onSearchStateChange).toHaveBeenCalledWith({
-        hello: 'there', // the instance didn't have the next props yet
+        hello: 'again', // the instance didn't have the next props yet
```

Comment:
```suggestion
        hello: 'again',
```

This comment was necessary, since this used to be stale state, but is now up-to-date state, which is correct